packages/plugins/src/plugins/index.ts

```diff
@@ -5,3 +5,4 @@ export { PrettierPlugin, PrettierPluginOptions } from './prettier.plugin.js';
 export { ReRehearsePlugin, ReRehearsePluginOptions } from './re-rehearse.plugin.js';
 export { GlintCheckPlugin } from './glint-check.plugin.js';
 export { GlintFixPlugin } from './glint-fix.plugin.js';
+export { TransformPlugin } from './transform.plugin.js';
```

Comment:
I think something like `ServiceInjectionsTransformPlugin` would be better :)

Reply:
totally agree